src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ComCaseloadSearchService.kt

```diff
@@ -72,28 +73,16 @@ class ComCaseloadSearchService(
     val onProbationCount = searchResults.count { it.isOnProbation == true }
     val inPrisonCount = searchResults.count { it.isOnProbation == false }
 
-    return ProbationSearchResult(
-      searchResults,
-      inPrisonCount,
-      onProbationCount,
-    )
-  }
-
-  fun getIneligibilityReasons(prisoner: Prisoner): List<String> {
```

Comment:
this was called by the caseload service - it now has it's own simpler implementation